platform/ui/src/components/Select/Select.css

```diff
@@ -0,0 +1,31 @@
+.customSelect__wrapper .customSelect__control {
+  @apply bg-black border-custom-blue shadow transition duration-300 border rounded w-full text-sm text-white leading-tight;
```

Comment:
I'm truly in love with the power Tailwind is giving us even to customize 3rd party libraries. It's awesome :)